app/css/Mindfulness.css

```diff
@@ -1,4 +1,15 @@
 /* Mindfulness.css */
+body {
+  font-size: 1rem; /* Base body text size; reflects user/system settings */
+}
```

Comment:
Shouldn't body font be defined in global.css?

Reply:
Resolved